delft/network.nix

```diff
@@ -1,4 +1,34 @@
-{
+let
+  makeMac = { ip, extra }: {
+    deployment = {
+      targetHost = ip;
+    };
+    macosGuest = {
+      enable = true;
+      network = {
+        interiorNetworkPrefix = "10.172.170"; #172="n", 170="x"
```

Comment:
what do `n` and `x` refer to in the comment?

Reply:
I was making a half-hearted attempt at spelling Nix

Reply:
oooo :D